evergreen/run-tests.sh

```diff
@@ -21,6 +21,4 @@ then
 fi
 
 dotnet clean "./MongoDB.EFCoreProvider.sln"
-
-
-dotnet test "./MongoDB.EFCoreProvider.sln" -e MONGODB_URI="${MONGODB_URI}" --runtime "${TARGET_RUNTIME}" --results-directory ./artifacts/test-results --logger "junit;LogFileName=TEST_{assembly}.xml;FailureBodyFormat=Verbose" --logger "console;verbosity=detailed"
```

Comment:
`--runtime "${TARGET_RUNTIME}"` is a wrong thing to do here, it should be `--framework` instead, but it also makes no sense as we have a single target for the project.